pkg/skaffold/initializer/init_test.go

```diff
@@ -79,11 +88,13 @@ func TestWalk(t *testing.T) {
 		{
 			description: "should return correct k8 configs and build files",
 			filesWithContents: map[string]string{
-				"config/test.yaml":  emptyFile,
-				"k8pod.yml":         emptyFile,
-				"README":            emptyFile,
-				"deploy/Dockerfile": emptyFile,
-				"Dockerfile":        emptyFile,
+				"config/test.yaml":    emptyFile,
```

Comment:
If files are alway empty, you could `use t.NewTempDir().Touch(name1, name2)` instead

Reply:
The `skaffold.yaml`s have contents in some of the tests.